package.json

```diff
@@ -132,9 +132,10 @@
     "cheerio": "^0.20.0",
     "codecov": "^1.0.1",
     "enzyme": "^2.0.0",
-    "eslint": "^2.4.0",
+    "eslint": "^3.1.1",
     "eslint-config-hapi": "^9.1.0",
-    "eslint-config-standard-react": "^2.2.0",
+    "eslint-config-standard-react": "^3.0.0",
+    "eslint-config-standard-jsx": "^2.0.0",
```

Comment:
Why did you introduce this?  I believe it's already a dep of eslint-config-standard-react.